service/src/main/java/bio/terra/pipelines/stairway/imputation/SubmitCromwellRunSetStep.java

```diff
@@ -148,8 +149,13 @@ public StepResult doStep(FlightContext flightContext)
     }
 
     // launch the submission
-    RunSetStateResponse runSetStateResponse =
-        cbasService.createRunSet(cbasUri, samService.getTspsServiceAccountToken(), runSetRequest);
+    RunSetStateResponse runSetStateResponse;
+    try {
+      runSetStateResponse =
+          cbasService.createRunSet(cbasUri, samService.getTspsServiceAccountToken(), runSetRequest);
```

Comment:
i dont think we should but still gonna ask.  Should we also put flight retries around the `CheckHealth` steps too?

Reply:
i mean probably yes? we don't want the flight dying if those fail for a transient reason

Reply:
you're right, we shouldn't add anything in the CheckHealth steps - it's built into the checkHealth() method to return a "not ok" status if the call throws an ApiException, and we are already retrying the step if we get a "not ok" status back from a health check. so we already are retrying, nothing to add there.